packages/web-components/src/global/_formation_overrides.scss

```diff
@@ -119,6 +119,7 @@
 .usa-select {
   background-size: rem-override(1.25rem);
   background-position: right rem-override(0.5rem) center;
+  margin-bottom: 0;
```

Comment:
The only things that should go into this formation override stylesheet are rem to pixel conversion for v3. Everything else should go into component specific stylesheets.